src/features/llm_analytics/components/LLMAnalyticsPlaceholder/LLMAnalyticsPlaceholder.tsx

```
@@ -0,0 +1,69 @@
+        <Text type="long-m">{i18n.t('llm_analytics.placeholder.you_can_also')}</Text>
+      </div>
+      <div>
+        <Trans
+          i18nKey="llm_analytics.placeholder.fill_bio"
+          components={{
+            icon: <User24 />,
+            lnk: (
+              <span
+                className={styles.clickableText}
+                onClick={() => goTo('/profile')}
+              ></span>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)